includes/class-wc-payments-features.php

```diff
@@ -29,6 +29,23 @@ public static function is_upe_enabled() {
 		return self::is_upe_legacy_enabled() || self::is_upe_split_enabled();
 	}
 
+	/**
+	 * Returns the "type" of UPE that ill be displayed at checkout.
```

Comment:
```suggestion
	 * Returns the "type" of UPE that will be displayed at checkout.
```
ope, forgot a "w" here 😅